src/algebra/pointwise.lean

```diff
@@ -620,6 +620,10 @@ lemma preimage_smul' [group_with_zero α] [mul_action α β] {a : α} (ha : a 
   (λ x, a • x) ⁻¹' t = a⁻¹ • t :=
 preimage_smul (units.mk0 a ha) t
 
+lemma preimage_smul_inv [group_with_zero α] [mul_action α β] {a : α} (ha : a ≠ 0) (t : set β) :
```

Comment:
```suggestion
lemma preimage_smul_inv' [group_with_zero α] [mul_action α β] {a : α} (ha : a ≠ 0) (t : set β) :
```
since the primed lemmas are for `group_with_zero`. This also might conflict with #9226, which adds lots more lemmas here and moves the existing ones around. Are you ok with merging that first?

Reply:
Sure, this can wait for #9226.